### PR TITLE
Minor cleanups in limiter code

### PR DESCRIPTION
**Pull request by fmahebert**
## Proposed changes

Various minor improvements

Edit: depends on #1658 

### Types of changes:

- [ ] Bugfix
- [ ] New feature

### Component:

- [ ] Code
- [ ] Documentation
- [ ] Build system
- [ ] Continuous integration

### Code review checklist

- [ ] The PR passes all checks, including unit tests and `clang-tidy`.
  For instructions on how to perform the CI checks locally refer to the [Dev
  guide on the Travis CI](https://spectre-code.org/travis_guide.html).
- [ ] The code is documented and the documentation renders correctly. Run
  `make doc` to generate the documentation locally into `BUILD_DIR/docs/html`.
  Then open `index.html`.
- [ ] The code follows the stylistic and code quality guidelines listed in the
  [code review guide](https://spectre-code.org/code_review_guide.html).

### CODE REVIEW

File: src/Evolution/DiscontinuousGalerkin/Limiters/MinmodType.hpp [hunk 1]
```diff
@@ -17,8 +17,7 @@ namespace Limiters {
 /// \see Limiters::Minmod for a description and reference.
 enum class MinmodType { LambdaPi1, LambdaPiN, Muscl };
 
-std::ostream& operator<<(std::ostream& os,
-                         const Limiters::MinmodType& minmod_type);
+std::ostream& operator<<(std::ostream& os, Limiters::MinmodType minmod_type);
```
Comment (nilsdeppe) [inline src/Evolution/DiscontinuousGalerkin/Limiters/MinmodType.hpp]: Should/could be `noexcept`. I'll let you decide whether you want to change it.

**Review by nilsdeppe — DISMISSED** [review]

File: src/Evolution/DiscontinuousGalerkin/Limiters/MinmodType.hpp (hunk 1 above)
Comment (fmahebert) [reply src/Evolution/DiscontinuousGalerkin/Limiters/MinmodType.hpp]: I have some test failures to address, so I'll fix this too :)

**Comment by fmahebert** [comment]
@nilsdeppe I've gone and marked _all_ the `operator<<`s as `noexcept` in another PR #1658 that this now depends on.

**Review by nilsdeppe — DISMISSED** [review]

File: tests/Unit/Evolution/DiscontinuousGalerkin/Limiters/Test_Weno.cpp [hunk 2]
```diff
@@ -293,8 +294,11 @@ void test_weno_tci_2d() noexcept {
   }
   ();
 
-  const std::array<double, 4> means_no_activation = {{-1., 1., -1., 1.}};
-  const std::array<double, 4> means_activation = {{-1., 1., -1., 0.9}};
+  const double eps = 100.0 * std::numeric_limits<double>::epsilon();
```
Comment (moxcodes) [inline tests/Unit/Evolution/DiscontinuousGalerkin/Limiters/Test_Weno.cpp]: explain the eps on the next few lines

**Review by moxcodes — DISMISSED** [review]
group code review with @fmahebert , @moxcodes primary

File: tests/Unit/Evolution/DiscontinuousGalerkin/Limiters/Test_Weno.cpp (hunk 2 above)
Comment (fmahebert) [reply tests/Unit/Evolution/DiscontinuousGalerkin/Limiters/Test_Weno.cpp]: I've removed the epsilon; fundamentally, there was no need for the slopes to be so close to the critical slope that triggers the limiter. There is now a larger value, and a new comment.

**Review by moxcodes — APPROVED** [review]

**Comment by fmahebert** [comment]
Rebased on develop after #1658 merged

**Review by nilsdeppe — APPROVED** [review]

**Pull request merged**